### ISSUE TITLE
Editting maniwani/templates/post-base.html

### ISSUE THREAD

**Issue by therealjr** [DangerOnTheRanger/maniwani#167]
I'm trying to add another field for new threads but any modifications I make to the file do not seem to appear. Am I correct in assuming that you must run docker-compose build/up to make changes take affect? If so, it doesn't seem to have any bearing on the generation of the file. What could I be doing wrong?

**Comment by DangerOnTheRanger** [comment]
If your branch includes #159 (it probably does, that was merged about 3 weeks ago) then most of the old Jinja templates don't do anything and should be removed. I've made #169 to track this problem. If you want to modify the new post and/or the new thread form, [frontend/src/components/PostForm.jsx](https://github.com/DangerOnTheRanger/maniwani/blob/f8f850c6a1efb7d43589ee227caa8f2f70e9fc1f/frontend/src/components/PostForm.jsx) has taken the place of the old `post-base.html` template. Keep in mind you'll need to run `docker-compose run maniwani update` every time you update/rebuild anything on the frontend so the JS is copied over to Minio.
